### PR TITLE
Rename _WIN64 define to BIT64

### PR DESCRIPTION
**Pull request by jkotas**
Avoids confusion with _WIN64 being defined even on Unix

### CODE REVIEW

File: src/pal/prebuilt/corerror/mscorurt.rc [hunk 1]
```diff
@@ -126,8 +126,8 @@ BEGIN
 	MSG_FOR_URT_HR(CORDBG_E_STATIC_VAR_NOT_AVAILABLE) "A static variable is not available because it has not been initialized yet."
 	MSG_FOR_URT_HR(CORDBG_E_CANT_SETIP_INTO_OR_OUT_OF_FILTER) "SetIP cannot leave or enter a filter."
 	MSG_FOR_URT_HR(CORDBG_E_CANT_CHANGE_JIT_SETTING_FOR_ZAP_MODULE) "JIT settings for ZAP modules cannot be changed."
-	MSG_FOR_URT_HR(CORDBG_E_CANT_SET_IP_OUT_OF_FINALLY_ON_WIN64) "SetIP is not possible because it would move EIP from within a finally clause to a point outside of one on WIN64 platforms."
-	MSG_FOR_URT_HR(CORDBG_E_CANT_SET_IP_OUT_OF_CATCH_ON_WIN64) "SetIP is not possible because it would move EIP from within a catch clause to a point outside of one on WIN64 platforms."
+	MSG_FOR_URT_HR(CORDBG_E_CANT_SET_IP_OUT_OF_FINALLY_ONBIT64) "SetIP is not possible because it would move EIP from within a finally clause to a point outside of one on WIN64 platforms."
```
Comment (AaronRobinsonMSFT) [inline src/pal/prebuilt/corerror/mscorurt.rc]: This should be `_BIT64`.

Comment (AaronRobinsonMSFT) [reply src/pal/prebuilt/corerror/mscorurt.rc]: Also, does the WIN64 in the message need to be updated?

File: src/pal/prebuilt/inc/corerror.h [hunk 2]
```diff
@@ -154,8 +154,8 @@
 #define CORDBG_E_STATIC_VAR_NOT_AVAILABLE EMAKEHR(0x131a)
 #define CORDBG_E_CANT_SETIP_INTO_OR_OUT_OF_FILTER EMAKEHR(0x131c)
 #define CORDBG_E_CANT_CHANGE_JIT_SETTING_FOR_ZAP_MODULE EMAKEHR(0x131d)
-#define CORDBG_E_CANT_SET_IP_OUT_OF_FINALLY_ON_WIN64 EMAKEHR(0x131e)
-#define CORDBG_E_CANT_SET_IP_OUT_OF_CATCH_ON_WIN64 EMAKEHR(0x131f)
+#define CORDBG_E_CANT_SET_IP_OUT_OF_FINALLY_ONBIT64 EMAKEHR(0x131e)
```
Comment (AaronRobinsonMSFT) [inline src/pal/prebuilt/inc/corerror.h]: `_BIT64`

File: src/pal/prebuilt/corerror/mscorurt.rc (hunk 1 above)
Comment (jkotas) [reply src/pal/prebuilt/corerror/mscorurt.rc]: Unclear. I am not able to find where these errors are used.

**Comment by janvorli** [comment]
You have accidentally renamed `CORDBG_E_CANT_SET_IP_OUT_OF_CATCH_ON_WIN64` too which is causing at least some of the CI failures.

**Comment by janvorli** [comment]
Or forgotten to update the pal/prebuilt/corerror/corerror.h, mscorurt.rc and inc/corerror.xml.

**Pull request merged**